### DISCUSSION TITLE
problem with not having vnc on one of my pi's

### DISCUSSION THREAD

**Discussion by shantamcbain** [any1/wayvnc#321] [General]
Hi I am not sure if this is the right forum for this.
I have one of my Rpi's that seems to be missing vnc. I just installed Wayland of the device. rpi-connect vnc on
✗ Screen sharing support is unavailable

I looked into installing a vnc. Of course, chatgpt said to install Realvnc. Docs pointed to wayvnc. Following the instructions on git failed to build the application past this point
shanta@hampi:~ $ meson build
ninja -C build

ERROR: Neither directory contains a build file meson.build.
WARNING: Running the setup command as `meson [options]` instead of `meson setup [options]` is ambiguous and deprecated.
ninja: Entering directory `build'
ninja: error: loading 'build.ninja': No such file or directory
shanta@hampi:~ $ ./build/wayvnc
-bash: ./build/wayvnc: No such file or directory
shanta@hampi:~ $ ls
aml        Desktop    Music     Public     wayvnc
Bookshelf  Documents  neatvnc   Templates
build      Downloads  Pictures  Videos
shanta@hampi:~ $ meson build

Can anyone point me to a solution to this issue? I am open to another vnc if that is simpler solution.

Thanks 
Shanta

**Comment by any1** [comment]
I recommend looking at the official raspberry pi forums for something like this. The "✗ Screen sharing support is unavailable" error probably means that you're not running a supported wayland compositor. You need either wayfire or labwc. Gnome or KDE won't work.
